docs/data_ensemble.md

```
@@ -54,6 +54,41 @@ Dry Contact information is available in the [EnvoyData.dry_contact_status](#pyen
 
 ```
 
+## Generator data
+
+Systems with an Enpower and a standby generator installed report generator data. Availability is signaled by the {py:attr}`pyenphase.const.SupportedFeatures.GENERATOR` supported feature flag.
+
+- Generator status (admin, operational relay state, admin mode, schedule state, generator present) is available in [EnvoyData.generator](#pyenphase.EnvoyData.generator), modeled by [EnvoyGenerator](#pyenphase.models.generator.EnvoyGenerator).
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)